packages/salesforcedx-vscode-lwc/src/testSupport/workspace/getCliArgsFromJestArgs.ts

```diff
@@ -16,7 +17,12 @@ export function getCliArgsFromJestArgs(
   testRunType: TestRunType
 ) {
   const cliArgs = ['--', ...jestArgs];
-  if (testRunType === TestRunType.DEBUG) {
+
+  const usePreviewJavaScriptDebugger = workspace
+    .getConfiguration('debug')
+    .get('javascript.usePreview');
+
+  if (testRunType === TestRunType.DEBUG && !usePreviewJavaScriptDebugger) {
     cliArgs.unshift('--debug');
```

Comment:
This is the gist of the fix. `--inspect-brk` is no longer necessary - https://github.com/microsoft/vscode/issues/99526#issuecomment-657114179 for the [new VS Code JavaScript debugger](https://github.com/microsoft/vscode-js-debug). (Note that the comment mentions a fallback. However it doesn't apply here because it looks up package.json scripts for `--inspect-brk` args).

Therefore we don't need the `--debug` args in `sfdx-lwc-jest` or `lwc-test` if users are using the new debugger. Launch configuration without `--debug` args would work with the new debugger. For user who has specified using the old debugger, we keep the same behavior.

cc @trevor-bliss @muenzpraeger

Reply:
Nice!